saleor/checkout/complete_checkout.py

```diff
@@ -249,6 +245,18 @@ def _create_line_for_order(
         discounts=discounts,
     )
 
+    undiscounted_unit_price = add_tax_to_undiscounted_price(
+        price=undiscounted_base_unit_price,
+        tax_rate=tax_rate,
+        prices_entered_with_tax=prices_entered_with_tax,
+    )
+
+    undiscounted_total_price = add_tax_to_undiscounted_price(
+        price=undiscounted_base_total_price,
+        tax_rate=tax_rate,
+        prices_entered_with_tax=prices_entered_with_tax,
+    )
+
```

Comment:
We need a test that is checking if a `undiscounted` price is with tax right now. Also, we need to ensure that for `create_order_from_checkout` the undiscounted prices are calculated with taxes too.